notebooks/4_SparseKernelMethods.ipynb

```diff
@@ -86,12 +86,26 @@
     "In sparse kernel methods, an approximate kernel is used in place of the full kernel. This approximate kernel is typically constructed according to the [Nystr&ouml;m approximation](https://en.wikipedia.org/wiki/Low-rank_matrix_approximations#Nystr%C3%B6m_approximation) [(Williams 2001)](http://papers.nips.cc/paper/1866-using-the-nystrom-method-to-speed-up-kernel-machines.pdf),\n",
     "\n",
     "\\begin{equation}\n",
-    "    \\mathbf{K} \\approx \\mathbf{\\tilde{K}}_{NN} = \\mathbf{K}_{NM} \\mathbf{K}_{MM}^{-1} \\mathbf{K}_{NM}^T,\n",
+    "    \\mathbf{K} \\approx \\mathbf{\\hat{K}}_{NN} = \\mathbf{K}_{NM} \\mathbf{K}_{MM}^{-1} \\mathbf{K}_{NM}^T,\n",
     "\\end{equation}\n",
     "\n",
     "Here, $M$ represents a subset of the $N$ total rows/columns of the kernel matrix, i.e. the kernel between a small **active set** than is selected with subsampling method, like farthest point sampling (FPS) [(Eldar 1997)](https://doi.org/10.1109/83.623193), or a CUR decomposition [(Imbalzano2018)](https://doi.org/10.1063/1.5024611), that is discussed in the [next notebook](5_CUR.ipynb)."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Precomputed X_train and X_test are centered and scaled.\n",
+    "# Unscale and uncenter them to build the sparse kernels\n",
+    "# so they can be directly compared to the full kernels,\n",
+    "# which are built on the raw X data\n",
```

Comment:
This comment is confusing. As a reader, it's unclear if we're un-centering because we *have* to or for demonstrative purposes.

Reply:
In the pre-loaded data, the full kernels are computed using uncentered/unscaled X, but the centered/scaled X are loaded. Therefore, if we want to be able to compare the full kernels with the sparse kernels that are constructed in the notebook, the sparse kernels must also be built on the uncentered/unscaled X data. Otherwise the effective kernel width is different between the two. This more has to do with the particularities of the automated variable loading than any general kernel-building practice. I will try to reword this to make that more clear.